src/api.js

```diff
@@ -1,9 +1,19 @@
 import express from "express";
+import bodyParser from "body-parser";
+import { postReview } from "./addReviewServer.js";
 
 const apiRouter = express.Router();
 
 apiRouter.get("/movies/:id", (req, res) => {
-	res.send(`movie with id of ${req.params.id}`);
+  res.send(`movie with id of ${req.params.id}`);
 });
 
+apiRouter.post(
+  "/movies/:movieId/reviews",
+  bodyParser.json(),
+  async (req, res) => {
```

Comment:
Is it necessary to wrap the function in an anonymous function? Or perhaps you can just pass the function itself as an argument?